packages/discord.js/src/structures/Poll.js

```diff
@@ -0,0 +1,106 @@
+'use strict';
+
+const { Collection } = require('@discordjs/collection');
+const { Routes } = require('discord-api-types/v10');
+const Base = require('./Base');
+const { PollAnswer } = require('./PollAnswer');
+const { DiscordjsError } = require('../errors/DJSError');
+const { ErrorCodes } = require('../errors/index');
+
+/**
+ * Represents a Poll
+ * @extends {Base}
+ */
+class Poll extends Base {
+  constructor(client, data, message) {
+    super(client);
+
+    /**
+     * The message that started this poll
+     * @name Poll#message
+     * @type {Message}
+     * @readonly
+     */
+
+    Object.defineProperty(this, 'message', { value: message });
+
+    /**
+     * The question text of this poll
+     * @type {string}
+     */
+    this.question = data.question.text;
```

Comment:
I'm torn on this. On one hand, I get why you'd raise the text up, but on the other... What if Discord ends up adding more fields to question in the future? 😅 
Leaving others to give their thoughts too, I'm not gonna block on this

Reply:
This field could be named `questionText` maybe, then if Discord adds more fields we could add a `question` object and deprecate `questionText`

Reply:
At that point just keeping it an object makes more sense 😄